addons/pos_sale/models/sale_order.py

```diff
@@ -59,6 +59,16 @@ def _compute_amount_to_invoice(self):
             order_amount = sum(order.sudo().pos_order_line_ids.filtered(lambda pol: pol.sale_order_line_id.is_downpayment).mapped('price_subtotal_incl'))
             order.amount_to_invoice -= order_amount
 
+    @api.depends('order_line.pos_order_line_ids')
+    def _compute_amount_invoiced(self):
+        super()._compute_amount_invoiced()
+        for order in self:
```

Comment:
```suggestion
        for order in self.filtered(lambda o: o.state in ['paid', 'done']):
```

We should only count it as `amount_invoiced` if the corresponding pos order is paid.

Reply:
Done, but a few lines below on the pos order